pkgs/servers/bge/default.nix

```diff
@@ -0,0 +1,51 @@
+{sbt,stdenv, fetchgit,jdk, makeWrapper}:
+
+with stdenv.lib;
+ stdenv.mkDerivation {
+    name = "bge";
+
+    meta = {
+     description = "Bitcoin Graph Explorer is a blockchain analysis server";
```

Comment:
Please consult the guidelines wrt meta information. For example, the package name should be omitted from the short description.